### PR TITLE
fix(ingest/bigquery): Fix BigQueryTableType enum accesses

### PR DESCRIPTION
**Pull request by asikowitz**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py [hunk 1]
```diff
@@ -19,7 +18,7 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
-class BigqueryTableType(Enum):
+class BigqueryTableType:
```
Comment (hsheth2) [inline metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py:21]: we could still use StrEnum?

**Review by hsheth2 — APPROVED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py (hunk 1 above)
Comment (asikowitz) [reply metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py:21]: Saw this, don't think it'll work here unless I change the queries -- that's what's actually failing, where I directly embed `BigQueryTableType.MATERIALIZED_VIEW` etc. in query text. If I keep as an Enum, then I have to change to `BigQueryTableType.MATERIALIZED_VIEW.value`, but I didn't see any value in keeping it as an enum when it's basically just a string const holder.

```
In [2]: from enum import Enum

In [3]: class Mine(str, Enum):
   ...:     A = "ONE"
   ...:     B = "TWO"
   ...: 

In [4]: Mine.A
Out[4]: <Mine.A: 'ONE'>

In [5]: Mine.A == "ONE"
Out[5]: True

In [6]: str(Mine.A)
Out[6]: 'Mine.A'

In [7]: str(Mine.A.value)
Out[7]: 'ONE'
```

**Review by jjoyce0510 — APPROVED** [review]

File: metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py (hunk 1 above)
Comment (hsheth2) [reply metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema.py:21]: ugh didn't realize that it overrode the `__str__`

yep let's leave it like this then

File: metadata-ingestion/tests/unit/test_bigquery_source.py [hunk 2]
```diff
@@ -464,7 +463,7 @@ def test_get_views_for_dataset(
             last_altered=bigquery_view_1.last_altered,
             comment=bigquery_view_1.comment,
             view_definition=bigquery_view_1.view_definition,
-            table_type=BigqueryTableType.VIEW,
+            table_type="VIEW",
```
Comment (mayurinehate) [inline metadata-ingestion/tests/unit/test_bigquery_source.py:466]: Not sure why this change was needed if we fixed the constants class already.

Comment (mayurinehate) [reply metadata-ingestion/tests/unit/test_bigquery_source.py:466]: this is test, then its okay.
